dll/shellext/netplwiz/netplwiz.spec

```diff
@@ -0,0 +1,14 @@
+1 stub AddNetPlaceRunDll
+2 stub PassportWizardRunDll
+3 stub PublishRunDll
+4 stub UsersRunDll
+5 stub ClearAutoLogon
+@ stdcall -private DllCanUnloadNow()
+@ stdcall -private DllGetClassObject(ptr ptr ptr)
+8 stub -private DllInstall
+9 stdcall DllMain(ptr long ptr)
+@ stub -private DllRegisterServer
+@ stub -private DllUnregisterServer
```

Comment:
I am not sure having `DllRegisterServer` and `DllUnregisterServer` as stub is a good idea.